tests/integration/metrics_tests/test_metrics.py

```diff
@@ -53,46 +55,47 @@ async def verify_endpoints(ops_test: OpsTest, app_name=DATABASE_APP_NAME):
 async def test_build_and_deploy(ops_test: OpsTest) -> None:
     """Build and deploy three units of MongoDB and one unit of TLS."""
     # no need to build and deploy charm if provided
-    mongodb_application_name = await ha_helpers.get_application_name(ops_test, DATABASE_APP_NAME)
-    if mongodb_application_name:
-        return
+    app_name = await get_app_name(ops_test)
+    num_units = NUM_UNITS
```

Comment:
```suggestion
    num_units = NUM_UNITS
```
remove this and use the constant on line 67

Reply:
done, thank you